### PR TITLE
Skeleton and Weapon Equip

### PR DESCRIPTION
**Pull request by garry911128**
Complete
1. Attack with Sword(no animation), Bomb(no throw)
2. Skeleton of Squirrel that can connect with Weapon(I intentionally placed the weapon in front of the hand so that the player's hitbox would not conflict with the weapon.)
3. Attack enemy.

### CODE REVIEW

File: Assets/Scripts/Core/Camera/CameraController.cs [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+            else
+            {
+                var targetPosition = target.position + Vector3.up * offset.y;
+                transform.rotation = Quaternion.RotateTowards(transform.rotation, Quaternion.LookRotation((targetPosition - transform.position).normalized), rotationSpeed * Time.deltaTime);
+            }
+        }
+
+        private void HandleInput()
+        {
+            // ����ƹ����ʿ�J
```
Comment (cuchaos) [inline Assets/Scripts/Core/Camera/CameraController.cs]: english?

File: Assets/Scripts/Core/Camera/CameraController.cs
Comment (kyynk) [inline Assets/Scripts/Core/Camera/CameraController.cs:1]: The namespace does not match the location.
If you want the file at this location, your namespace should be `Core.Camera`.
If you want the namespace to be `Entities.Player`, you need to move this file under the Player folder.

File: Assets/Scripts/Core/Camera/CameraController.cs [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+            lookDelta.x = Input.GetAxis("Mouse X");
+            lookDelta.y = Input.GetAxis("Mouse Y");
+
+            // �ˬd�O�_Ĳ�o Lock On
```
Comment (cuchaos) [inline Assets/Scripts/Core/Camera/CameraController.cs]: same above

**Review by kyynk — CHANGES_REQUESTED** [review]
**The namespace should match the folder name.**

P.S. Please remove all `Chinese` comments, they will cause encoding issues.

File: Assets/Scripts/Core/Camera/CameraController.cs (hunk 2 above)
Comment (garry911128) [reply Assets/Scripts/Core/Camera/CameraController.cs]: fix

File: Assets/Scripts/Core/Camera/CameraController.cs
Comment (garry911128) [reply Assets/Scripts/Core/Camera/CameraController.cs:1]: fix

File: Assets/Scripts/Core/Camera/CameraController.cs (hunk 1 above)
Comment (garry911128) [reply Assets/Scripts/Core/Camera/CameraController.cs]: fix

File: Assets/Scripts/KeyboardInput/KeyboardInputManager.cs [hunk 3]
```diff
@@ -35,6 +38,23 @@ protected override void CalculateDpadAxis()
...
+
+                // �T�O Y �b���s�A�קK�v�T���⫫����V
+                forward.y = 0;
+                right.y = 0;
+                forward.Normalize();
+                right.Normalize();
+
+                // ���a�b�ഫ���@�ɶb
+                axis = (forward * axis.z + right * axis.x).normalized;
+            }
```
Comment (kyynk) [inline Assets/Scripts/KeyboardInput/KeyboardInputManager.cs]: remove `Chinese` comments

**Review by kyynk — CHANGES_REQUESTED** [review]

**Review by kyynk — APPROVED** [review]

**Pull request merged**